dev/docker-compose.dev.yml

```diff
@@ -25,6 +25,7 @@ services:
             - AUTH_HOST=auth
             - MESSAGE_BUS_HOST=redis
             - DATASTORE_DATABASE_HOST=postgres
+            - DATASTORE_LOG_LEVEL=CRITICAL
```

Comment:
Remove uninteresting log ouput of the datastore reader when running tests with the `-s` flag